README.md

```diff
@@ -1,18 +1,70 @@
-Longhorn Upgrade Responder
+Upgrade Responder
 ========
 
-## Building
+## Overview
+Upgrade Responder enables developers to know how many users are using their applications. 
+This project is GDPR compliant since it doesn't store any information that can identify the user, not even IP address.
 
-`make`
+For example, longhorn use Upgrade Responder project to create the public metric dashboard at [https://metrics.longhorn.io/](https://metrics.longhorn.io/)
 
+## Usage
```

Comment:
## How it works
1. Upgrade Responder server is set up at an application-specified endpoint.
2. A response JSON file (e.g. xxxx) is provided to the server as the response to the incoming request.
    1. In Kubernetes deployment, the response JSON is normally configured as a Config Map and can be easily swapped out when a new version is available.
3. The application now can make HTTP requests in a certain format to the application-specified endpoint periodically, and get the information about the latest available version.
    1. Server will expect each instance of the application makes a request **once per hour**.
4. Once the server received a request, it will:
    1. Check where the request is coming from.
    2. Remove the IP from the request
    3. Store the following information into an InfluxDB table:
        * Application Version
        * Country and City of the request originated from
        * X-Request-ID which is a random string per request that is generated by the client.
        * Other information that's can be helpful to identify the upgradability, e.g. Kubernetes version.